app/assets/stylesheets/debug_frame.scss

```diff
@@ -0,0 +1,17 @@
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
```

Comment:
I would suggest to move all variables to variables.css.scss file and include in each file where you need